src/gluonts/nursery/temporal_hierarchical_forecasting/model/cop_deepar/_estimator.py

```diff
@@ -172,6 +172,9 @@ def __init__(
         return_forecasts_at_all_levels: bool = False,
         naive_reconciliation: bool = False,
         dtype: Type = np.float32,
+        impute_missing_values: bool = False,
+        imputation_method: Optional[MissingValueImputation] = None,
```

Comment:
Why do we need two parameters for that, shouldn't `imputaion_method=None` mean no imputation?

Reply:
Here I am following how `DeepAR` does it: https://github.com/awslabs/gluonts/blob/dev/src/gluonts/mx/model/deepar/_estimator.py#L181C13-L181C13